runtime-image/rh-jdk17-overrides.yaml

```diff
@@ -1,7 +1,7 @@
 schema_version: 1
 
 packages:
-  content_sets_file: content_sets_rhel8.yml
+      content_sets_file: content_sets_rhel9.yml
```

Comment:
indentation looks suspicious... Should it be 2 spaces instead?

Reply:
Fixed, thanks.